src/vs/workbench/contrib/webview/browser/webviewWindowDragMonitor.ts

```diff
@@ -32,5 +32,11 @@ export class WebviewWindowDragMonitor extends Disposable {
 				onDragEnd();
 			}
 		}));
+
+		this._register(DOM.addDisposableListener(targetWindow, DOM.EventType.DRAG, (event) => {
+			if (event.shiftKey) {
+				onDragEnd();
+			}
```

Comment:
we could change this to re-disable the webview when the user releases shift. 

```
			if (event.shiftKey) {
				onDragEnd();
			} else {
                                onDragStart();
                        }
```

Reply:
Yes I think that makes sense. Otherwise you have to restart dragging if you want to open the dragged resource in an editor instead of dropping it into the webview

Reply:
@mjbvz I've made some updates to address the feature that was requested. I noticed you tagged this with the April 2024 milestone and I would love to address any change requests/concerns prior to that deadline. 

And thank you for your hard work!